dotnet-efcore/EntityFrameworkCoreVsDapper/Tests/PersonRepositoryDapperUnitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using EntityFrameworkCoreVsDapper.Repositories;
+using Assert = Xunit.Assert;
+
+namespace Tests
+{
+    public class PersonRepositoryDapperUnitTests
+    {
+        private PersonsRepositoryDapper repository = new PersonsRepositoryDapper();
+
+        [Fact]
+        public void When_Quering1000Persons_ThenReturn1000Objects()
+        {
+            var queryResults = repository.Get_1000_Persons();
+
+            Assert.True(1000 == queryResults.Count());
+        }
+
+        [Fact]
+        public void When_Quering10000Persons_ThenReturn10000Objects()
+        {
+            var queryResults = repository.Get_10000_Persons();
+
+            Assert.True(10000 == queryResults.Count());
```

Comment:
Since the result is a List you can use the List.Count property directly rather than the IEnumerable LINQ Count function.